zio-http-gen/src/main/scala/zio/http/gen/openapi/EndpointGen.scala

```diff
@@ -1077,7 +1078,10 @@ final case class EndpointGen(config: Config) {
           ),
         )
       case JsonSchema.Null    => throw new Exception("Null query parameters are not supported")
-      case JsonSchema.AnyJson => throw new Exception("AnyJson query parameters are not supported")
+      case JsonSchema.AnyJson => {
+        // throw new Exception("AnyJson query parameters are not supported")
```

Comment:
why did we previously throw exception?
Am I breaking some other validation now?

Reply:
@987Nabil any idea regarding this one?

Reply:
1. Please no `{}` block for a case handling :)
2. The exception was that there is no handling implemented. Not that there can't be one
3. `None` seems odd. Can you explain?

Reply:
1. sure, will do
2. the exception refers to query parameters, so that was weiird.
3. the method is `def schemaToCode(…): Option[Code.File]`, there is no need to generate a file for `Json`. It's just an import. So `None` was obvious to me.